### ISSUE TITLE
Global consistency tests: how was the tolerance/range created or calculated?

### ISSUE THREAD

**Issue by samuele-mattiuzzo** [IATI/IATI-Website-Tests#141]
This is one for @hayfield and @dalepotter (and whoever else you guys think might know the answer for this)

Consistency tests often fail because of discrepancy between the range the tests think it's safe to assume a value will be contained in and the actual value. 

I need to understand how those ranges were calculated (and I need an ELI5 type of explanation and something more than what we have in the docs/code comments) otherwise I'll nuke those tests before end of this week.

Thanks

**Comment by dalepotter** [comment]
Happy to have a quick call to see if I can cast any light on this - I'll message you offline to try to find a  suitable time.

**Comment by samuele-mattiuzzo** [comment]
We can discuss it here rather than a private call

**Comment by hayfield** [comment]
For the ones about minimum publisher / file / activity count: they're hard coded as being some value ~10% below whatever it was at the time. They should be updated on a periodic basis to provide new minimums. This detects sudden disappearance of large amounts of data.

For the percentage-difference ones: ideally all systems would show the same numbers, though systems could take a day or several to update, so some discrepancy is present as organisations register and files are published. A small percentage (1-5%) allows for systems to take a little while to update, but flags if significant discrepancy occurs.

**Comment by samuele-mattiuzzo** [comment]
Thanks Hayden!

**Comment by samuele-mattiuzzo** [comment]
I think this is actually all that's needed on our end (@Ocre42 to thumbsup and eventually close if happy as well), as the idea of the periodic manual update is all I needed to understand (as opposed to what I assumed was a fixed valued that came after a deep and intense conversation between Dale, Hayden and Imogen with some deep space complex maths I am completely devoid of) :P
